about/edge-policy-console.md

```diff
@@ -84,7 +84,7 @@ The **system info** box shows information such as IDs, system statuses, and othe
 The **logging level** box has two buttons, `set log level to debug` and `set log level to info` which controls how verbose your logging is.
 
 <!--
-To see the logs, follow [these instructions](/reference/debugging/#a-namelogsagetting-access-to-the-ambassador-logs).
+To see the logs, follow [these instructions](/reference/debugging/#review-ambassador-logs).
 -->
```

Comment:
I think you meant to also remove the `<!--` and `-->`?

Reply:
So they were actually IN the doc before I edited it, should I remove those? Does it make it invisible?

Reply:
Yeah, they make what's between them invisible.  When we deleted the old debugging page, I'd added them to hide the sentence instead of deleting it.